Components/Card/Card.css

```diff
@@ -0,0 +1,46 @@
+
+body{
```

Comment:
The responsiveness belongs to the card. Not to the body. Imagine you develop the card component in isolation. It can be one or many on the page.